libs/api-spec/api-spec.yml

```diff
@@ -2471,3 +2524,40 @@ components:
                 type: string
                 example: after meals
       description: Request body to update prescription
+    ExistingMedicalRecord:
+      content:
+        application/json:
+          schema:
+            type: object
+            x-examples:
+              example-1:
+                patient_status: string
+                symptoms: 'fever, sneeze'
+                icd10_code: A750
+                suggestions: full rest 10 days
+                patient_id: 516a962a-802f-41f5-a2e0-c315146ad2d4
+                polyclinic_id: 1

```

Comment:
Prefer delete this because it already declared in properties

Reply:
Applied https://github.com/ALTERA-CAPSTON-41/backend/pull/68/commits/dbdb94f0b1c19b7a7091901f8cb6ac6ff754bae7.